src/main/server.ts

```diff
@@ -289,26 +294,41 @@ const unwrapArgs = function (sender: WebContents, frameId: number, contextId: st
 
 const isRemoteModuleEnabledImpl = function (contents: WebContents) {
   const webPreferences = contents.getLastWebPreferences() || {}
-  return webPreferences.enableRemoteModule != null ? !!webPreferences.enableRemoteModule : false
+  return (webPreferences as any).enableRemoteModule != null ? !!(webPreferences as any).enableRemoteModule : false
 }
 
-const isRemoteModuleEnabledCache = new WeakMap()
+const isRemoteModuleEnabledCache = new WeakMap<WebContents, boolean>()
 
 export const isRemoteModuleEnabled = function (contents: WebContents) {
-  if (!isRemoteModuleEnabledCache.has(contents)) {
+  if (hasWebPrefsRemoteModuleAPI && !isRemoteModuleEnabledCache.has(contents)) {
     isRemoteModuleEnabledCache.set(contents, isRemoteModuleEnabledImpl(contents))
   }
 
   return isRemoteModuleEnabledCache.get(contents)
 }
 
+export function enable(contents: WebContents) {
+  if (hasWebPrefsRemoteModuleAPI) {
+    throw new Error("The `enable` API is available only for electron >= 14.0.0")
+  }
```

Comment:
```suggestion
```

We should allow older versions to use the `enable()` API, so that an app on Electron 13 can upgrade to @electron/remote@2.0.0, and start using the new `enable()` API, before upgrading to Electron 14.